image/verify/src/verifier.rs

```diff
@@ -455,10 +463,31 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
             raise_err!(FmcDigestMismatch)
         }
 
-        // TODO: Perform following Address checks
-        // 1. Load address is a valid ICCM Address
-        // 2. Entry Point is a valid ICCM Address
-        // 3. Entry Point is within the image
+        // TODO: Perform following Address check
+        // Entry Point is within the image
```

Comment:
What about alignment check? For both address and entry-point?

Reply:
Added.